src/main/java/uk/gov/ons/census/fwmt/jobservice/converter/impl/HouseholdConverter.java

```diff
@@ -41,9 +41,12 @@ public CaseRequest convert(CreateFieldWorkerJobRequest ingest) {
 
     Contact contact = new Contact();
     contact.setName(ingest.getAddress().getPostCode());
-    contact.setOrganisationName(ingest.getContact().getOrganisationName());
-    contact.setPhone(ingest.getContact().getPhoneNumber());
-    contact.setEmail(ingest.getContact().getEmailAddress());
+    if (!StringUtils.isEmpty(ingest.getContact())) {
```

Comment:
ingest.getContact() isn't a string, so should just check if its null

Reply:
Changed to null check